### ISSUE TITLE
Beacon destroyed by miners

### ISSUE THREAD

**Issue by SillySerotonin** [TangoTek/TekTopia-Community#962]
Miners did not stop covering or mining when my beacon was in their way. The iron pyramid remains intact, only the beacon is gone. I checked my miners inventories and it is not there either.

I would expect the tunnel to become inactive like when there is a lava fall above the tunnel.

I was playing on the patreon server when it happened.
My town hall is at -3348 68 -12333.

**Comment by GoldenFurredFox** [comment]
Sadly, I do not believe that this is an issue to be posted to the github, unless you're suggesting that they disable miners from breaking certain blocks. As of right now, Miners only break every block in their path, they don't question what block it is, and they only pick up vanilla ores that they mine.

**Comment by TraxKB** [comment]
I agree with GreenBean, I think this is an unfortunate accident, not an issue.

**Comment by GoldenFurredFox** [comment]
I think that the staff might grant you another beacon if you tell them that your miner broke it...
I think the real question here, though, is why you had a beacon anywhere underground, especially near Y=11.

**Comment by SillySerotonin** [comment]
Yea, I know it was really dumb to have put it there, I was kinda on autopilot for normal gameplay when I placed it. 
I am suggesting that Miners question the crafted/rare blocks, like beacons, similarly to how they see lava as dangerous. 
I have no background it coding but the more I think about it, I understand why miners are set up the way they are, so I guess just disregard this report.
I'm not worried about the beacon itself, another player suggested I report it as a bug.
